src/repositories/order_repository_test.go

```diff
@@ -0,0 +1,228 @@
+package repositories
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/ELITE-Kinoticketsystem/Backend-KTS/src/.gen/KinoTicketSystem/model"
+	kts_errors "github.com/ELITE-Kinoticketsystem/Backend-KTS/src/errors"
+	"github.com/ELITE-Kinoticketsystem/Backend-KTS/src/managers"
+	"github.com/ELITE-Kinoticketsystem/Backend-KTS/src/models"
+	"github.com/ELITE-Kinoticketsystem/Backend-KTS/src/samples"
+	"github.com/ELITE-Kinoticketsystem/Backend-KTS/src/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateOrder(t *testing.T) {
+	order := &model.Orders{
```

Comment:
Technally you can exchange this with one from the samples package, but it's fine this time

Reply:
Done